lib/elixir/lib/module.ex

```diff
@@ -1789,6 +1789,18 @@ defmodule Module do
     :ets.insert(set, {key, value, line})
   end
 
+  defp put_attribute(_module, :on_load, value, line, set, bag) do
+    try do
+      :ets.lookup_element(set, :on_load, 3)
+    catch
+      :error, :badarg ->
+        :ets.insert(set, {:on_load, value, line})
+        :ets.insert(bag, {:attributes, :on_load})
+    else
+      _ -> raise ArgumentError, "the @on_load attribute can only be called once per module"
```

Comment:
I think we should say "can only be **set** once per module", wdyt?

Reply:
👌